.github/workflows/release-announcements.yml

```
@@ -14,6 +14,11 @@ jobs:
         run: |
           echo "topics=$(curl -s https://nf-co.re/pipelines.json | jq -r '.remote_workflows[] | select(.full_name == "${{ github.repository }}") | .topics[]' | awk '{print "#"$0}' | tr '\n' ' ')" | sed 's/-//g' >> $GITHUB_OUTPUT
 
+      - name: get description
+        id: get_topics
+        run: |
+          echo "description=$(curl -s https://nf-co.re/pipelines.json | jq -r '.remote_workflows[] | select(.full_name == "${{ github.repository }}") | .description' >> $GITHUB_OUTPUT
```

```suggestion
        id: get_description
        run: |
          echo "description=$(curl -s https://nf-co.re/pipelines.json | jq -r '.remote_workflows[] | select(.full_name == "${{ github.repository }}") | .description')" >> $GITHUB_OUTPUT
```
this is a bug that will be fixed in the new template update. If you don't plan to release before that, i.e. in a week or so, you can leave it as is.